lib/pure/asynchttpserver.nim

```diff
@@ -35,7 +35,7 @@
 ##     server.listen Port(8080)
 ##     while true:
 ##       if server.shouldAcceptRequest():
-##         asyncCheck server.acceptRequest(cb)
+##         waitFor server.acceptRequest(cb)
 ##       else:
 ##         poll()
```

Comment:
I'm not sure why the `poll` is here tbh. Can we just remove it?

```suggestion
##         asyncCheck server.acceptRequest(cb)
```

Reply:
the poll is there to avoid exceeding maxFds, https://github.com/nim-lang/Nim/pull/16586 clarifies this